### DISCUSSION TITLE
Ajout de l'alias "d" pour la commande daily

### DISCUSSION THREAD

**Discussion by cyril-marpaud** [Crownicles/Crownicles#759] [Suggestions fermées]
L'alias "d" est actuellement disponible (inutilisé par d'autres commandes) et il est plus court et pratique que "da" (l'alias actuel pour la commande daily).

Il serait donc pratique d'ajouter l'alias "d" à la commande daily.

Voire la PR fermée https://github.com/DraftBot-A-Discord-Adventure/DraftBot/pull/758.

**Comment by BastLast** [comment]
Les alias n'existent plus, je ferme la suggestion
